Venmo/src/test/java/com/braintreepayments/api/VenmoClientUnitTest.java

```diff
@@ -433,6 +437,29 @@ public void tokenizeVenmoAccount_whenVenmoNotInstalled_forwardsExceptionToListen
         verify(braintreeClient).sendAnalyticsEvent("pay-with-venmo.app-switch.failed");
     }
 
+    @Test
+    public void tokenizeVenmoAccount_whenFallbackToWebTrueAndAppInstalled_appSwitchIsSuccessful() {
```

Comment:
Feels like we are missing the other cases like `whenFallbackToWebTrueAndAppNotInstalled` - are those tested somewhere else?

Reply:
Added some additional tests here: https://github.com/braintree/braintree_android/pull/911/commits/0a8e84284f11fd1308c528fdd1c5f2c0cb0c75be